tests/client/api_events/test_milestones_latest_event.c

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#include <unity/unity.h>
+#include "client/api/events/node_event.h"
+#include "client/api/events/sub_milestone_latest.h"
+#include "events_test_config.h"
+#include "string.h"
```

Comment:
for better practice, put headers in different groups.
make sure the include syntax follow the rule: https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html

```suggestion
#include <unity/unity.h>
#include <string.h>

#include "client/api/events/node_event.h"
#include "client/api/events/sub_milestone_latest.h"
#include "events_test_config.h"
```

Reply:
Done.